lib/pages/contacts/contact_details_page.dart

```diff
@@ -59,7 +84,7 @@ class _ContactDetailsPageState extends State<ContactDetailsPage> {
         children: <Widget>[
           Avatars.buildContactAvatar(memoryImage: contact.avatar, radius: 40.0),
           _buildName(),
-          // _buildShortcutsRow(),
+          widget.isProfile ? Container() : _buildShortcutsRow(),
```

Comment:
```suggestion
          if (!widget.isProfile) _buildShortcutsRow(),
```
No need for empty containers